content/en/community/hackathons/sessions/baby-steps/content/Setup_Nginx.md

```diff
@@ -0,0 +1,118 @@
+
+     ```bash
+     $ git clone https://github.com/unikraft/app-nginx nginx
+     ```
+
+     Enter the `nginx/` directory:
+
+     ```bash
+     $ cd nginx/
+
+     $ ls -F
+     config-qemu-aarch64  config-qemu-x86_64  fs0/  kraft.yaml  Makefile  Makefile.uk  README.md  run-qemu-aarch64.sh*  run-qemu-x86_64.sh*
+     ```
+
+  1. While inside the `nginx/` directory, create the `.unikraft/` directory:
+
+     ```bash
+     $ mkdir .unikraft
+     ```
+
+     Enter the `.unikraft/` directory:
+
+     ```bash
+     $ cd .unikraft/
+     ```
+
+  1. While inside the `.unikraft` directory, clone the [`unikraft` repository](https://github.com/unikraft/unikraft):
+
+     ```bash
+     $ git clone https://github.com/unikraft/unikraft unikraft
+     ```
+
+  1. While inside the `.unikraft/` directory, create the `libs/` directory:
+
+     ```bash
+     $ mkdir libs
+     ```
+
+  1. While inside the `.unikraft/` directory, clone the library repositories in the `libs/` directory:
+
+     ```bash
+     $ git clone https://github.com/unikraft/lib-nginx libs/nginx
+
+     $ git clone https://github.com/unikraft/lib-musl libs/musl
+
+     $ git clone https://github.com/unikraft/lib-lwip libs/lwip
+     ```
+
+  1. Get back to the application directory:
+
+     ```bash
+     $ cd ../
+     ```
+
+Ok, we have now cloned everything we need.
+We are ready to jump into `$make menuconfig` to configure our application by selecting all of the relevant libraries.
+First, remember to go through the `Architecture` and `Platform` menus and select the relevant options, just as we did for the `helloworld` application!
+After you have done that, you can scroll all the way down in the `Library Configuration` menu and select `libnginx` from the list (by pressing the `'Y'` key).
+All of its dependencies should also be selected automatically after you do this.
+
+![libnginx select](/community/hackathons/sessions/baby-steps/images/menuconfig_select_libnginx.png)
+
+You will then need to further configure the `libnginx` library by hitting `Enter` on the entry, and then selecting the `Provide main function` option
+
+![libnginx select main](/community/hackathons/sessions/baby-steps/images/menuconfig_select_libnginx_main.png)
+
+As a final configuration part, we need to explicitly specify which filesystem we are going to use. We can do this by leveraging the `vfscore` internal library:
+
+![select vfscore](/community/hackathons/sessions/baby-steps/images/menuconfig_select_vfscore.png)
+
+Make sure to check automatic mounting and the `9PFS` filesystem:
+
+![select vfscore fs](/community/hackathons/sessions/baby-steps/images/menuconfig_select_vfscore_fs.png)
+
+Finally, we can now build the `Nginx` application by running the classic `make` command:
+
+```bash
+$ make -j($nproc)
+```
+
+The compilation process will take some time to finish since the application, together with its libraries, is much more bigger than `helloworld`.
```

Comment:
```suggestion
The compilation process will take some time to finish since the application, together with its libraries, is much bigger than `helloworld`.
```